### PR TITLE
Ma/storybook snapshots

### PR DESCRIPTION
**Pull request by markusahlstrand**
Think I got it working now. It makes a snapshot of all the button variants for now. If we add the tags to the stories it will automatically snapshot all variants.

### CODE REVIEW

File: packages/lib/src/stories/Button.stories.ts [hunk 1]
```diff
@@ -42,7 +42,8 @@ export const Default: Story = {
     disabled: false,
     loading: false,
     variant: 'primary'
-  }
+  },
+  tags: []
```
Comment (markusahlstrand) [inline packages/lib/src/stories/Button.stories.ts:46]: We could remove the tags here.

File: playwright.config.ts [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  },
+
+  /* Configure projects for major browsers */
+  projects: [
+    {
+      name: 'chromium',
+      use: { ...devices['Desktop Chrome'] }
+    }
+
+    // {
```
Comment (markusahlstrand) [inline playwright.config.ts:42]: Skipped the other browsers for now. Should we add we add more?

File: tests/storybook.spec.ts [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+});
+
+const visualStories = filterStories(Object.values(manifest.entries));
+
+visualStories.forEach((story) => {
+  test(story.id, async ({ page }, meta) => {
+    await navigate(page, storybookUrl, meta.title);
+    const upstreamScreenshot = `${meta.title}-upstream-${process.platform}.png`;
+
+    await page.waitForTimeout(1000);
```
Comment (frankjonatanschultz) [inline tests/storybook.spec.ts:29]: is this really needed? 🤔

Comment (markusahlstrand) [reply tests/storybook.spec.ts]: This was for debugging to see why it didn't render :) Turned out I was using the wrong port..

**Pull request merged**